gnomad/sample_qc/pipeline.py

```diff
@@ -291,16 +305,40 @@ def infer_sex_karyotype(
             normal_ploidy_cutoff=normal_ploidy_cutoff,
             aneuploidy_cutoff=aneuploidy_cutoff,
         )
+        group_by_expr = None
+
+    if chr_x_frac_hom_alt_expr is not None:
+        logger.info(
+            "Including cutoffs for the fraction of homozygous alternate genotypes (hom-alt/(hom-alt + het)) on "
+            "chromosome X."
+        )
+        chr_x_frac_hom_alt_expr = ploidy_ht._chr_x_frac_hom_alt
+        chr_x_frac_hom_alt_cutoffs = get_chr_x_hom_alt_cutoffs(
+            ploidy_ht,
+            chr_x_frac_hom_alt_expr,
+            f_stat_cutoff=f_stat_cutoff,
+            group_by_expr=group_by_expr,
+            cutoff_stdev=normal_chr_x_hom_alt_cutoff,
+        )
+
+    else:
+        chr_x_frac_hom_alt_expr = None
```

Comment:
I don't think you need this since it already defaults to None?

Reply:
oops, yup, thank you. I think I reorganized things and accidentally left it.